Botusanu Darius-Gabriel/lab 4, 3.5/main.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+#include <stdlib.h>
+/*I haven't got a clue how to do the prime part...*/
```

Comment:
Here's something that will help
https://lmgtfy.com/?q=prime+number+algorithm+c&s=g